Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.11-alpine3.8 as builder
```

Comment:
I'd prefer we start using our internal image for that. But for now this should do.